share/picongpu/examples/KelvinHelmholtz/etc/picongpu/8_pipe.cfg

```diff
@@ -0,0 +1,121 @@
+# Copyright 2013-2022 Rene Widera, Felix Schmitt, Axel Huebl, Franz Poeschel
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                 docs/TBG_macros.cfg.
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="00:15:00"
+
+TBG_devices_x=2
+TBG_devices_y=2
+TBG_devices_z=2
+
+TBG_gridSize="192 512 32"
+TBG_steps="400"
+
+TBG_periodic="--periodic 1 1 1"
+
+
+#################################
+## Section: Optional Variables ##
+#################################
+
+TBG_PIC_config="\
+{ \
+  \"rank_table\": \"hostname\", \
+  \"adios2\": { \
+    \"engine\": { \
+      \"parameters\": { \
+        \"QueueLimit\": \"1\", \
+        \"InitialBufferSize\": \"4Gb\", \
+        \"Profile\": \"Off\", \
+        \"Threads\": \"!TBG_coresPerGPU\", \
+        \"MarshalMethod\": \"BP5\" \
```

Comment:
This is an undocumented parameter of SST which switches on a serialization backend that does not require the `InitialBufferSize` parameter. It does not properly send `vector<string>`-type attributes, so use with care.